proto/regen/group/v1alpha1/tx.proto

```diff
@@ -239,6 +247,10 @@ message MsgVoteRequest {
 
     // metadata is any arbitrary metadata to attached to the vote.
     bytes metadata = 4;
+
+    // try_exec will try to execute the proposal immediately 
+    // after the vote, if set to true.
+    bool try_exec = 5;
```

Comment:
shall we use string here `--exec={no, try}` to enable `must` option in the future?

Reply:
@robert-zaremba what about using an enum instead?

Reply:
yes, good idea.